terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1077,7 +1077,7 @@ def sq(*words):
                                           '${aws_vpc.gitlab.cidr_block}'
                                           if split_tunnel else
                                           all_ipv4
-                                      ],
+                                      ],  # noqa: E122 https://github.com/PyCQA/pycodestyle/issues/372
```

Comment:
Is this still necessary after the flake8 update? What the relationship between flake8 and pycodestyle?